### DISCUSSION TITLE
GitHub repo provides free access to paid fonts of my authorship

### DISCUSSION THREAD

**Discussion by ArcanaFoundry** [community/community#34521] [Other Feature Feedback, Questions, & Ideas]
I was recently made aware of a project that distributes two fonts I created a couple of years back as their own. As an independent designer, this has a perceivable impact on my small business.
What can I do, if anything? 😥

**Comment by fire-eggs** [comment]
Contact them, and if they don't comply, find a copyright lawyer.  If it is a Github project, report to Github.
